tests/AppBundle/Controller/MembershipControllerTest.php

```diff
@@ -255,7 +251,7 @@ public function testPinInterests()
 
     public function testPinInterestsPersistsInterestsForNonActivatedAdherent()
     {
-        /* @var Adherent $adherent */
+        /** @var Adherent $adherent */
```

Comment:
As I known, PHPDoc require tow stars, also in inline mode.

https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#55-examples